otter/convergence/service.py

```diff
@@ -237,8 +229,8 @@ def convergence_succeeded(scaling_group, group_state, servers, now):
         yield Effect(
```

Comment:
Non-blocking question that has nothing to with this PR - I think I'm just blanking - how is the group's active servers updated if the server bounces back from ERROR to ACTIVE?  This seems to only update the cache if the group was ACTIVE in the first place.

We specifically have a test for recovering from ERROR, and it passes, I just can't figure out what that mechanism is right now.  I'm just missing something obvious probably.